Buddies/Models/User.swift

```diff
@@ -223,6 +229,7 @@ class LoggedInUser : User {
                     location: data["location"] as? GeoPoint,
                     shouldSendJoinedActivityNotification: shouldSendJoinedActivityNotification,
                     shouldSendActivitySuggestionNotification: shouldSendActivitySuggestionNotification,
+                    filterSettings: data["filter_settings"] as? [Int] ?? FilterSearchBar.defaultSettings,
```

Comment:
Any particular reason we use an array to store the settings? Seems like an object might be more clear

Reply:
It would be a `[String : Int]` with keys. I'll change this?